Islandora-Devops/claw-playbook
# Text extraction

# What does this Pull Request do?

Adds text extraction module to playbook.

# What's new?
With text extraction in place images with OCR will have that text extracted and put into an editable media.
Any Original File with an Original File tag will also have text extracted into an editable media

# How should this be tested?
After playbook is spun up create a node tagged as both Image and Digital document.
Add image media (containing test) tagged as Original File.
Extracted Text media should be created and attached to node.

Create another node and attach a media tagged with Original File and with a media type of application/pdf.
Extracted Text media should be created and attached to node.

# Interested parties
 @Islandora-Devops/committers
